src/displayapp/screens/settings/SettingPineTimeStyle.cpp

```diff
@@ -0,0 +1,332 @@
+  lv_obj_align(btnNextBar, lv_scr_act(), LV_ALIGN_IN_RIGHT_MID, -15, 0);
+  lv_obj_set_style_local_bg_opa(btnNextBar, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LV_OPA_30);
+  lv_obj_set_style_local_value_str(btnNextBar, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, ">");
+  lv_obj_set_event_cb(btnNextBar, event_handler);
+
+  btnPrevBar = lv_btn_create(lv_scr_act(), nullptr);
+  btnPrevBar->user_data = this;
+  lv_obj_set_size(btnPrevBar, 60, 60);
+  lv_obj_align(btnPrevBar, lv_scr_act(), LV_ALIGN_IN_LEFT_MID, 15, 0);
+  lv_obj_set_style_local_bg_opa(btnPrevBar, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LV_OPA_30);
+  lv_obj_set_style_local_value_str(btnPrevBar, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, "<");
+  lv_obj_set_event_cb(btnPrevBar, event_handler);
+
+  btnNextBG = lv_btn_create(lv_scr_act(), nullptr);
+  btnNextBG->user_data = this;
+  lv_obj_set_size(btnNextBG, 60, 60);
+  lv_obj_align(btnNextBG, lv_scr_act(), LV_ALIGN_IN_RIGHT_MID, -15, 80);
+  lv_obj_set_style_local_bg_opa(btnNextBG, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LV_OPA_30);
+  lv_obj_set_style_local_value_str(btnNextBG, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, ">");
+  lv_obj_set_event_cb(btnNextBG, event_handler);
+
+  btnPrevBG = lv_btn_create(lv_scr_act(), nullptr);
+  btnPrevBG->user_data = this;
+  lv_obj_set_size(btnPrevBG, 60, 60);
+  lv_obj_align(btnPrevBG, lv_scr_act(), LV_ALIGN_IN_LEFT_MID, 15, 80);
+  lv_obj_set_style_local_bg_opa(btnPrevBG, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LV_OPA_30);
+  lv_obj_set_style_local_value_str(btnPrevBG, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, "<");
+  lv_obj_set_event_cb(btnPrevBG, event_handler);
+
+  btnReset = lv_btn_create(lv_scr_act(), nullptr);
+  btnReset->user_data = this;
+  lv_obj_set_size(btnReset, 60, 60);
+  lv_obj_align(btnReset, lv_scr_act(), LV_ALIGN_CENTER, 0, 80);
+  lv_obj_set_style_local_bg_opa(btnReset, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LV_OPA_30);
+  lv_obj_set_style_local_value_str(btnReset, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, "Rst");
+  lv_obj_set_event_cb(btnReset, event_handler);
+
+  btnRandom = lv_btn_create(lv_scr_act(), nullptr);
+  btnRandom->user_data = this;
+  lv_obj_set_size(btnRandom, 60, 60);
+  lv_obj_align(btnRandom, lv_scr_act(), LV_ALIGN_CENTER, 0, 0);
+  lv_obj_set_style_local_bg_opa(btnRandom, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LV_OPA_30);
+  lv_obj_set_style_local_value_str(btnRandom, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, "Rnd");
+  lv_obj_set_event_cb(btnRandom, event_handler);
+}
+
+SettingPineTimeStyle::~SettingPineTimeStyle() {
+  lv_obj_clean(lv_scr_act());
+  settingsController.SaveSettings();
+}
+
+bool SettingPineTimeStyle::Refresh() {
+  return running;
+}
+
+void SettingPineTimeStyle::UpdateSelected(lv_obj_t *object, lv_event_t event) {
+  uint8_t valueTime = settingsController.GetPTSColorTime();
+  uint8_t valueBar = settingsController.GetPTSColorBar();
+  uint8_t valueBG = settingsController.GetPTSColorBG();
+    
+  if((object == btnNextTime) && (event == LV_EVENT_PRESSED)) {
```

Comment:
`event == LV_EVENT_CLICKED` could be tested at the beginning of the function, so it wouldn't need to be duplicated in each if statement. CLICKED is preferred over PRESSED.